language-server/modules/langserver-core/src/test/resources/completion/function/chainCompletion3.json

```diff
@@ -714,4 +758,4 @@
       "insertTextFormat": "Snippet"
     }
   ]
-}
+}
```

Comment:
new line

Reply:
check other places also

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/27492/commits/101c1d115241d121429b93cab8ca4f57d1e72c85